README.md

```diff
@@ -8,32 +8,35 @@ Configure this ML Model service as a modular resource on your machine with a [Je
 
 ## Requirements
 
-A NVIDIA Jetson Orin board with the following installed:
+A NVIDIA Jetson Orin board or other machine with an NVIDIA GPU with the following installed:
 
-1. [Jetpack 5](https://developer.nvidia.com/embedded/jetpack)
+1. [CUDA](https://developer.nvidia.com/cuda-toolkit), [Jetpack 5, or Jetpack 6](https://developer.nvidia.com/embedded/jetpack)
```

Comment:
If you are just using the "other machine with an NVIDIA GPU" then you don't need JP.

Reply:
Agreed: that's why I said you need CUDA, JP5, or JP6. How would you prefer this to be clarified?